content/source/docs/enterprise/api/team-access.html.md

```diff
@@ -16,7 +16,9 @@ The team access APIs are used to associate a team to permissions on a workspace.
 | :----- | :------------- |
 | GET | /team-workspaces |
 
-### Parameters
+### Query Parameters
+
+These are standard URL query parameters. Prefix the first one with a `?`, and use `=` between the parameter name and the value. When using more than one parameter, separate them with `&`. Special characters in the parameter names must be URL-encoded. (`[` becomes `%5B`, etc.)
 
 - `?filter[workspace][id]` (`string: <required>`) - The workspace ID for which to list the teams with access
```

Comment:
I wish we could restructure the filters so that they were not URL query parameters at all and avoid the need for `?` and `&` and also the need for URL encoding.

Reply:
Yeah :/ but that would mean making every filterable GET into a POST, right? Well, I'm the wrong person to figure out if that's feasible.  Anyway, I don't think we use any other query parameters, since the complex endpoints all seem to be POSTs anyhow. 

I'm not married to the `?`, at any rate.

Reply:
Filtering will almost always usually be conducted on a `GET` request so it's preferable to have it as part of the query parameters.

If we're only using query parameters for filtering and including it might be nice to separate those in to sections as they could be detailed (linked back to overview) that gives people a better understanding. I know the including parameter definitely needs some definition to go with it.